### ISSUE TITLE
https://yelp.github.io/pyleus/  404!

### ISSUE THREAD

**Issue by l3rnao** [Yelp/pyleus#182]
hi, all
sorry to find that the document page 404!
pls check that!

**Comment by dan-blanchard** [comment]
I just noticed this myself, but apparently this project is now deprecated.  I would recommend people consider switching to [streamparse](https://github.com/Parsely/streamparse), since we still actively maintain that (and have developer preview releases that support Storm 1.0).

**Comment by l3rnao** [comment]
@dan-blanchard  oops! so cool! and is the streamparse use MessagePack to serializer?

**Comment by dan-blanchard** [comment]
@l3rnao, yes, but it's not super straightforward yet, because that's only possible in the developer preview releases of 3.0.  They're actually quite stable, but we haven't pushed out 3.0 final yet because there are some little things I would like to have in place for the release.  You can install the 3.0 preview with `pip install streamparse==3.0.0.dev3`.  To use it, you need to follow the steps outlined [here](https://github.com/Parsely/streamparse/issues/247#issuecomment-213426132).

**Comment by l3rnao** [comment]
@dan-blanchard thank you soooo much.  have a nice day : )
